src/cascadia/TerminalApp/defaults.json

```diff
@@ -30,41 +29,53 @@
             "guid": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
             "name": "Windows PowerShell",
             "commandline": "powershell.exe",
-            "hidden": false,
-            "startingDirectory": "%USERPROFILE%",
-            "closeOnExit": "graceful",
+            "icon": "ms-appx:///ProfileIcons/{61c54bbd-c2c6-5271-96e7-009a87ff44bf}.png",
             "colorScheme":"Campbell Powershell",
+
+            // The following settings are the same across all profiles

```

Comment:
It implies that there is some magic here in these two profiles that _makes_ it the same. Can we just dispense with the commentary here and leave a note in the user setttings file.